doc/source/getting_started/install.rst

```diff
@@ -344,15 +363,19 @@ top-level :func:`~pandas.read_html` function:
 XML
 ^^^
 
+Can be managed with the ``xml`` extra.
+
 ========================= ================== =============================================================
 Dependency                Minimum Version    Notes
 ========================= ================== =============================================================
-lxml                      4.5.0              XML parser for read_xml and tree builder for to_xml
+lxml                      4.6.3              XML parser for read_xml and tree builder for to_xml
 ========================= ================== =============================================================
 
 SQL databases
 ^^^^^^^^^^^^^
 
+Can be managed with the ``postgresql, mysql, sql-other`` extras, depending on required sql compatibility.
```

Comment:
Would be good to add another column to specify which dependency is captured with which label.